src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -709,8 +713,9 @@ protected function processRepositories()
 				$entityArgument = $boundEntity;
 
 			} else {
-				$entityArgument = new Code\PhpLiteral('"%entityName%"');
-				$this->postCompileRepositoriesQueue[$boundManagers[0]][] = [ltrim($originalDef->getClass(), '\\'), $originalServiceName];
+				$repository = ltrim($originalDef->getClass(), '\\');
```

Comment:
Now, generated repository services uses EntityLocator for repository to entity class mapping.
DIC generated class must not be overwritten anymore (%entityName%).